infra/pr_helper.py

```diff
@@ -100,9 +100,13 @@ def main():
     if email:
       if is_known_contributor(content_dict, email):
         # Checks if the email is verified.
-        message += (
-            f'{pr_author} is either the primary contact or is in the CCs list '
-            f'of [{project_path}]({project_url}).<br/>')
+        if verified:
+          message += (
+              f'{pr_author} (verified) is either the primary contact or '
```

Comment:
Can we just do something like this:

```pypi

verified_marker = ' (verified') if verified else ''
message = f'....{verified_marker}'
```

This way we avoid some duplication.

Reply:
For unverified email, should we continue checking PR commit history? Currently, we don't check history for verified email.